src/main/java/rx/Observable.java

```diff
@@ -12,21 +12,32 @@
  */
 package rx;
 
-import java.util.*;
-import java.util.concurrent.*;
-
-import rx.annotations.*;
-import rx.exceptions.*;
+import rx.annotations.Beta;
+import rx.annotations.Experimental;
+import rx.exceptions.Exceptions;
+import rx.exceptions.OnErrorNotImplementedException;
 import rx.functions.*;
 import rx.internal.operators.*;
 import rx.internal.producers.SingleProducer;
-import rx.internal.util.*;
-import rx.observables.*;
+import rx.internal.util.RxRingBuffer;
+import rx.internal.util.ScalarSynchronousObservable;
+import rx.internal.util.UtilityFunctions;
+import rx.observables.BlockingObservable;
+import rx.observables.ConnectableObservable;
+import rx.observables.GroupedObservable;
 import rx.observers.SafeSubscriber;
-import rx.plugins.*;
-import rx.schedulers.*;
+import rx.plugins.RxJavaObservableExecutionHook;
```

Comment:
Please make sure the imports stay with `*` because it is a source of merge conflict otherwise.